backend/backends/moss_tts_nano_backend.py

```
@@ -0,0 +1,180 @@
+MOSS-TTS-Nano backend implementation.
+
+Wraps NanoTTSService from moss-tts-nano for zero-shot voice cloning.
+0.1B parameters, 48 kHz stereo output, 20 languages, CPU-friendly.
+WeTextProcessing is optional (Chinese text normalization); if absent,
+the tts_robust_normalizer_single_script fallback is used instead.
+"""
+
+import asyncio
+import logging
+from pathlib import Path
+from typing import List, Optional, Tuple
```

Ruff (0.15.10)

[warning] `typing.List` is deprecated, use `list` instead

(UP035)